Mathlib/GroupTheory/GroupAction/Defs.lean

```diff
@@ -496,14 +496,13 @@ theorem one_smul (b : α) : (1 : M) • b = b :=
 
 /-- `SMul` version of `one_mul_eq_id` -/
 @[to_additive "`VAdd` version of `zero_add_eq_id`"]
-theorem one_smul_eq_id : ((· • ·) (1 : M) : α → α) = id :=
-  funext <| one_smul _
+theorem one_smul_eq_id : (((1 : M) • ·) : α → α) = id := funext <| one_smul _
```

Comment:
I think this works?
```suggestion
theorem one_smul_eq_id : ((1 : M) • · : α → α) = id := funext <| one_smul _
```

Reply:
Yes, well-spotted. I noticed it too but only halfway through so I didn't manage to find all the ones I had already suboptimally fixed.